### ISSUE TITLE
Connecting to multiple networks

### ISSUE THREAD

**Issue by wei2912** [sarenji/pyrc#17]
I think this would be a good feature to add but it may not fit the goal of `pyrc` being a slim and concise library. Any opinions?

**Comment by sarenji** [comment]
Possibly. Perhaps you can just define two bots and have them connect in two separate threads?

**Comment by wei2912** [comment]
Good idea. Is there a way to do so without having seperate threads, though?
